CHANGELOG.md

```diff
@@ -4,6 +4,277 @@ All notable changes to this project will be documented in this file.
+  - Fix(GraphQL): Fix between filter bugs ([#6822][])
+  - Fix(GraphQL): Fix panic error when we give null value in filter connectives. ([#6707][])
+  - Fix(GraphQL): Remove extra fields when querying interfaces ([#6596][])
+  - Fix(GraphQL): disallowing field names with as ([#6579][])
+  - Fix(GraphQL): Fix object Linking with `hasInverse` ([#6557][])
+  - Fix(GraphQL): Fix cascade with auth query when RBAC is false ([#6444][])
+  - Fix(GraphQL): Generate correct schema when no orderable field in a type ([#6456][])
+  - Fix(GraphQL): Fix  restoreStatus query with query variables ([#6414][])
+  - Fix(GraphQL): Fix for deletion on interfaces with no non Id field ([#6387][])
+  - Fix(GraphQL): don't generate orderable enum value for list fields ([#6392][])
+  - Fix(GraphQL): Fix introspection completion bug ([#6385][])
+  - Fix(GraphQL): Extend int64 range to 64-bit numeric values and adds input coercing and
+   validation for integers. ([#6275][])
+  - Fix(GraphQL): Remove auth error from mutation. ([#6329][])
+  - Fix(GraphQL): Fix query rewriting for auth delete when deleting types with inverse field. ([#6350][])
+  - Fix(GraphQL): incorrect generatedSchema in updateGQLSchema ([#6349][])
+  - Fix(GraphQL): Link xids properly if there are duplicate xids within the same add request. ([#6265][])
+  - Fix(GraphQL): Fix internal error when doing GraphQL schema introspection after drop all ([#6268][])
+  - Fix(GraphQL): Fixes unexpected fragment behaviour ([#6228][])
+  - Fix(GraphQL): Fix order and offset in auth queries. ([#6221][])
+  - Fix(GraphQL): Linking of xids for deep mutations ([#6172][])
+  - Fix(GraphQL): Don't reserve certain queries/mutations/inputs when a type is remote. ([#6055][])
+  - Fix(GraphQl): Allow case insensitive auth header for graphql subscriptions. ([#6141][])
+  - Fix(GraphQl): Panic Fix when subscription expiry is not present in jwt. ([#6129][])
+  - Fix(GraphQL): Fix bug in custom resolver, now body need not have all the fields. ([#6054][])
+  - Fix(GraphQL):  Disallow  Subscription typename. ([#6077][])
+  - Fix(GraphQL): Fixes wrong query parameter value for custom field URL ([#6074][])
+  - Fix(GraphQL): Fixes panic in update mutation without set & remove ([#6073][])
+  - Fix(GraphQL): Fix auth rewriting for nested queries when RBAC rule is true. ([#6053][])
+  - Fix(GraphQL): Fix getType queries when id was used as a name for types other than ID ([#6130][])
+
+- Core Dgraph
+  - Fix(ludicrous mode): Handle deletes correctly ([#6773][])
+  - Fix(Zero): Fix how Zero snapshots and purge works ([#7096][])
+  - Fix: Check for nil ServerCloser in shutdown handler ([#7048][])
+  - Fix(health): Update health only after upserting schema and types ([#7006][])
+  - Fix(worker): Flush the stream writer on error (DGRAPH-2499) ([#6609][])
+  - Fix(export): don't return an error if there was no GraphQL schema ([#6815][])
+  - Fix pointer misalignment ([#6795][])
+  - stability(zero): add mmaped keyCommit map ([#6804][])
```

Comment:
@aman-bansal , the keyCommit map is no longer mmaped. We use go memory for it. 
The change is: instead of using go map, we are using custom B-tree. (ref: https://github.com/dgraph-io/ristretto/commit/d04b4c27e6326fd311bbbd48d6c2aac8ee4ef0a7)